library/src/main/java/com/chuckerteam/chucker/api/ChuckerInterceptor.kt

```diff
@@ -70,6 +75,8 @@ class ChuckerInterceptor @JvmOverloads constructor(
             if (io.isPlaintext(buffer)) {
                 val content = io.readFromBuffer(buffer, charset, maxContentLength)
                 transaction.requestBody = content
+                transaction.operationName =
+                    Gson().fromJson(content, HttpRequestBody::class.java).operationName
```

Comment:
This will be really inefficient :(
You're going to recreate a `Gson` instance for every request.
Please use a `JsonConvertor.getInstance()`.